.github/workflows/check.yml

```diff
@@ -8,54 +8,52 @@ concurrency:
   group: ${{ github.ref }}
   cancel-in-progress: true
 jobs:
-  test-pdflatex-linux-windows:
+  test-pdflatex:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        ruby: [ '3.1', '3.2', '3.3' ]
+        os:
+          - ubuntu-latest
+          # TODO: re-enable test on MacOS once we have a way to install TeXlive
+          #- macos-latest
```

Comment:
@koppor FYI: Action which you introduced doesn't [seem to support macOS](https://github.com/reitzig/ltx2any/actions/runs/8215641921/job/22469329750):

```
run echo "date=$(date -I)" >> $GITHUB_OUTPUT
date: option requires an argument -- I
```

IIRC, that's a standard problem you run into because macOS doesn't come with GNU tools by default. 

Happy to accept a PR that re-introduces testing on macOS!